src/main/java/com/google/api/codegen/clientconfig/ClientConfigSnippetSetRunner.java

```diff
@@ -29,11 +29,10 @@
   private static final String SNIPPET_RESOURCE_ROOT =
       ClientConfigSnippetSetRunner.class.getPackage().getName().replace('.', '/');
 
-  @SuppressWarnings("unchecked")
   @Override
-  public GeneratedResult generate(ElementT element,
-      String snippetFileName,
-      CodegenContext context) {
+  @SuppressWarnings("unchecked")
+  public GeneratedResult generate(
+      ElementT element, String snippetFileName, CodegenContext context) {
```

Comment:
Once @pongad's PR is in master and merged here, I think this will be a moot point.